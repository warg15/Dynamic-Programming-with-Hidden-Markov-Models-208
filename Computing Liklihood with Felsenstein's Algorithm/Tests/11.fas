>Grey-crowned-crane
TAGCGTCTCACAGATAAACTATGTCCATTGCGCGAATGTATGATAGCTTCGATTTCGAGTCTTTTCCCCTCCTCCCTTACCAAACGTCCTAGGACGTGTAAACTGTGTAACTTGAAGCAGACTTATGTACCTGTCAAATGTACTGAGTAACACCCGAAACAAGATTATGAGCGTGATCCAATGGTAGATAGATTTCGGGC
>Golden-collared-manakin
TAGCATCTCACAGATAAACTATGTTCAGTGCGCGAATGTATGATAGCTTCGATTTCGAGTCTTTCCCGCGCCTCCCTTACCAAACCTCCTAGGACGTGTGAACAGTGTAAGTTTAAGCAGACTGATGTCCCTGTCAAATACACGGAGAAGCACCGGAAACAAGATTATAAGGGGGAGCCAATGGTAGATAGGTTTCGGGC
>White-throated-tinamou
TAGAATCTCACGGATAGACTTGATCCATTGCGCGGATGCATGAGAGCTTAGAGTTCAAGTCTTTCGCGCTTCTCTCTTACCACGTGTCTTAGGGCGTGACAATTGCGTAAGTTAAGACAGACTGCTGTCCCCATCAAATAGACGGAGCAACACCAGATGCCAGATTTCGAGGGGGATCAAGTGGTAGACAAATTTCGGGC
>White-tailed-eagle
TAGCATCTCACAGATAAACTATGTTCATTGCGCGAATGTATGATAGCTTCGATTTCGAGTCTTTTCCGCTGCTCTCTTACCAAACCCCCTAGGACGTGTAAACTGTGCAAGTTGAAACAGACTGATGTCCCTGTTAAACATATTGAGTAACACCGGAAACAGGATTATGAGGGTGACCCAGTGGTAGTTAGATTTCGCGC
>Kea
TAGCATTTCACAGATAATCTACGTTCAATTCGCGAATGTATGATAGCTTCAATTTCGAGTCTTTTCCGTTCCTCTTTTACCAAAGCCCCTAGAACGTGTATATGGTCTGAGTTGAAGCAAACTGATGTCCCTGTCAAATATACTGAAAAACACCGGAAACGAGATTTTGAGGGTGATCCAATGGTAGATAGATTTCGCGC
>White-tailed-tropicbird
CTGCATTTCACAGATAAACTATGTTCATTGCGCGAATGTATGATAGCTTCGATTTCGAGTCTTTTCCGCTCCTGTCTTACCAAGCCTCATAGGACGTGTAAACTGTGCAAGTTGAAGCAGACTGGTATCCGTGTCAAATATACTAAGTAACACCGGTAGCAAGATTATGAGGGTGATCCAATGGTAGATAGATTTAGGGC
>American-crow
TGGCATCTCATATATAATCTATATTAAATGCGTGAATGTATGATTGCTTCGATTTCGAGTCTTTCCCGCTCCTCTCTTACCAGACCTCCCAGGACGTGTAAACAGTATAAGTTTACGCAGACTGGTGTCCCTGTCAAATACACTGAGAAACACCAGAAACAAAATTATAAGGGTGAGCGATGGGTAGATAAATTTCGGGC
>Red-legged-seriema
TAGCATCTCACAGATAAACTGTGTTCAATGGGCAAATGTATGATAGCTTCGATTTCGAATCTTTTCCGCTCCTCTCTTACCAAACCCCCTAGGACGTGTGAACTGTGTAAGTTGAAGCAGTCTGATGTCCCTGTCAAATATACTGAGAAACTCCGGAAACAAGATTATGAAGGTGATCCAATGGTAGATAGATTTCGGGC
>Turkey-vulture
TAGCATCTCACAGATAAACTATGTTCATTGCGCGAATGTATGATAGCTTCGATTTCGAGTCTTTTCCGCTCCTCTCTTACCAAACCCCCTAGGACGTGTAAACTGTGTAAGTTGAAGCAGACTGATGTCCCTGTCAAATATACTGAGTAACACCGGAAACAAGATTATGAAGGTGATCCAATGGCAGATAGATTTCGGAC
>Peregrine-falcon
TAGCATCTCACACATAAATTATGTTCAATACGCGAATGTATGGTAGCTTTGATTTCGAGTCTTCTCCACTACTCTCTTACCAAACCCCCTAAGACGTGTAAACAGTGTAAGTTGAGGCAGACTGTTATCCCTGTCAAATATTCTGAGAAACACCGGAAACAAGATTACGAGGCTGATCCATTGGTTGATAGATTTTGGGC
>Northern-fulmar
TAGCATCCCACAAATAAACTATGTTCATTGCGCGAATGTATGATAACTTCGATTTCGAGTCTTTTCCGCTCCTCTCTTACCAAACCTCCTAGGACGTGTAAACTGTGTAAGTTGAAGCAGACTGATGTCCCGGTGAAATATACTGAGTAACACCGGAAACAAGATTATGAGGGTGATCCAATGGTAGATAGATTTCGGGT
>Chuck-wills-widow
TAGCACCTCACAGATAAACTGTGTTCATTGTGCGAATATATGATAGCTTCGTTTTCGAGTCTTTGCCGCTCCTCTCTTACCAAACCTCCTAGGACGTGCAAATTGTGTGAGTTGAAGCGGACTGGTGTCCCTGTCAAATATACTGAGTAACACCGGAAACAAGCTTATGAGGATGATCCAATGGTAGATAGATTTGGGGC
>Common-ostrich
TAGACTCTTGCAGATAGATTTTCTCCAATCCGCGGATGTATGAGAGCTTCGAGTTCGAGTCTTTTCCGCTCGTCTCTTACCAAATGTCTTAGGACGTGAGAATTGTGTAAGTTGAAGCACATTGATTTCCCCGTCAAATAGACTGAGTAACACCAAACATAAGATTGTGACGGTGATCGAATGATGGACAGATTTCGGGC
>Great-cormorant
TAGCGTCTCGCAGATAAACTATGTTTATTGCGCGAATGTATGATAGTGTCGATTTCGAGTTTTTTCCGCTCCTCTCTTACCAAACCTCCTAGGACGTGTAAACTGTGTAAGTTGAAGCAGACTGGTGTCCCGCTAAAATATACTGAGTAACACCGGAAACAAGATTATGAGGGTGATCCAATGGTAGATAGATTTCGGGC
>Adelie-penguin
TAGCAACTCACAGATAAACTATGTTGATTGCGCGAATGTATGATAGCTTCGATTTCGAGTCTTTTCCGCTCCTCTCTTACCAAACCTCCTAGGACGTGTAAACTTTGTAAGTTGAAGCAGACTGATGTCCCGGTGAAATATACTGAGTAACACCGGTATCAAGATTATGTGGGTGACCCAATGGTAGATAGATTTCGGGC
>Zebra-finch
TAGTATCTCATATATAATCTATATTCAATGCGAGAATGTATGATTGCTTCGATTTCGAGGCTTTCCCGCTCCTCTCTTACCAAACCTCCCAGGATATGCAAACAGTATAAGTTTACGAAGACAGATGTCCCGGTCAAATACACCGAGAAACACCGAAAAAACAATTATAAGGGTGAGCGATTGGTGGATAGATTTCGGGC
>Little-egret
TAGCATCTCACAAATAAACTATGTCCATTGCGCGAATGTATGATAGCTTCGATTTCGAGTCTTTTCCGCTCCTCTCTTACTAAACCTCCTAGGACATCTAAACTTTGTAAGTTGAAGCAGACTGATGTCCCAGTGAAATATACTGAGTAACACCGAAAACAAGATCATGAGGGTGATCCAATGCTAAATAGATTTCGGGC
>Pigeon
TAGCATCTGACAGATAAACTGTGTTCATTGCGCGAATCTATGATAGCTTCGATTTCGAGTCTTTTTCGCTCCTCTCTTACGAAACCTCCTAGGACGTGTATACTGCGTAAGTGGGAGCAGACTGATATCCCTGTCAAAGATACTGAGTAGCACCGGAAACAACATTATTGAGGTGATCCAATGGTAGATAGATTTCGGGC
>Red-throated-loon
TAGCATCACACAGATAAACTATGTTCATTGCGCGAATGTATGATAGCTTCGATTTCGAGTCTTTTCCGCTCCTCTCTTACCAAACCTCCTAGGACGTGTAAACTGTGTGAGTTGAAGCAGATTGATGTCCCTGTGAAATATCCCGAGTAACACCGGAAACAAGATTATGAGGGTGATCCAATGGTACATAGATTTCGGGC
>Turkey
TAGAGTCTCACAGTTGGACTATGTCCGGTGCGCGAATGTATAAGAGGTTCGATTTCGAATCTTTTCCGCCCTTCTCTTACCAAACCTTACAGTACGCGTGAACTGTGTAAGTTTAGGCAGGCTAATGTCCCTGTCAGATATACTGAGTAACTCTGGAAGCAAAATTATCAAGGTACTCGAAAGATAGATAAATTTTGGGC
>Speckled-mousebird
TGGTATCCCACAAATAAACAATGTTCATTGCGCGAATGTATGCTAGCTTCGATTTTGACTCTTTTCCGCTGATCTCTTATCAAACCCCCTAGGACGTGTAAACTGTGTGAGTTGAAGCAGACCGATGTCCCTGTCAAATGTACTAAGTAACACCGGAAACAAGATCAAGAAGGTCATCAAATGGTAGATAGCTTTCGGGC
>Brown-mesite
TAGCATCTCACACATAAACTATGTTCATTGCGCAAATGTATGATAGCTTGGATTGCGAGTCTTTTCCGCTCCTCTCTTACCAAACCTCCTGGGACGTGTACACTGTATAAGATGAAGCAGACTGATGTCCCTGTCAAAAATACTGAGTAACACCAGAAAAAAGATTATGAGGGGGACTCAGCGGTAGATAGATTTTGGGC
>Red-crested-turaco
TAACGTCTCACAGATAAACTATGTTCATTGTGCGAATGTATGGTAACTTCGATTTCGAGTCTTTTCCGCTCTTCTCCTACCACACCTCCTAGGACGTGTAAACTGTGTAAGTTGGATCAGACTGATATCCCTGTCAAATATACTGAGTAACATCGGAAACAAGATTATAAGGGTGATCCAATGGTAGATACATTCCGGGC
>Sunbittern
TCGCATCTCACAGATAAACTATGTTCATTGCGCGAACGTATGATAGCTTCGATTACGAGTCTTTTCCGCTCCTCTCTTACCAAACCTCCCAGGACGTGTAAGCTATGTAGGTTGAAGCATACTGATGTCCCTGTCAAATATACTGAGTAACACCGAAAACACGCTTATGAGGGTGATCCAACGGTAGATAGATTTCGGGG
>Dalmatian-pelican
TAGCATGACACAAATAAACTATGTTCATTGCACGAATGTATGATAGCTTCGATTTCGAGTCTTTTCCGGTCCTCTCTTACCAAACCTCCTAGGACGTGTAGACTGTGTAAGTTGAAGCAGACTGATGTCCCGGTGAAATATACTGAGTAACACCGGAAACAAGATTATTAGGGTGATCCAATGGTAGATAGATTTCGGGC
>Cuckoo-roller
TAGCATCTCACAGATAAACTATGTTCATTGTGCGAATGTATGATAGCTTCGATTTCGAGTCTTTTCCGCTCCTCTCTTACCAAACCCCCTAGTACGTGTAAACTGTGCAACTTGAAGCGAACTGATGTCCCTGTCAGATATACCGAGTAACACTTGAAAGAAGATTACGAGGGTGATCCAATGGTAAATAGATTTCCGGC
>Bald-eagle
TAGCATCTCACAGATAAACTATGTTCATTGCGCGAATGTATGATAGCTTCGATTTCGAGTCTTTTCCGCTGCTCTCTTACCAAACCCCCTAGGACGTGTAAACTGTGCAAGTTGAAACAGACTGATGTCCCTGTTAAACATATTGAGTAACACCGGAAACAGGATTATGAGGGTGACCCAGTGGTAGTTAGATTTCGCGC
>Emperor-penguin
TAGCATCTCACAGATGAACTATGTTCATTGTGCGAATGTATGATAGCTTCGATTTCGAGTCTTCTCCGCTCCTCTCTTACCAAACCTCCTAGGACGTGTAAACTGTGTAAGTTGAAGCAGACTGATGTCCCGGTGAAATATACTGAGTAACACCGGAATCAAGATTATGAGGGTGACCCAATGGTAGATAGATTTCGGGC
>Crested-ibis
TAGCATCTCACAAATGAACTATGTTCATTGCATGAAGGTATGCTAGCTTCGATTTCGAGTCTTTTCCGTTCCTCTCTTACCAAACCTCCTAGGACGTGTAAACTGTATAAGTTGAAGCCGACTGATGTCCCGGTAAAATACATTGAGTAACACCGGAAACAAGATTATGAGGGTGATCCAATGGTAGATAGATTTCGGGC
>Killdeer
TAGCATCTCACAGATAAACCATGTTCATTGCGCGAATGTATGATAGCTTCGATTTCGATTCTTTTCCGCTCCTCTCTTACGAAACCTCCTAGGACGTGTAAACTGTGTAAGTTGAAGCAGACTGATGTCCCTGTCAAATATACTGGGAAATACCGAAAACAATATTATGAAGGTGATACAGTGATAAATAGATTTCGGGC
>Medium-ground-finch
TAGCATCTCATATAGAATCTATATTCAATGTGTGAATGTATGATTGCTTCGATTTCGAATCTTTCCCGCTCCTCTCTTACCAAACCTCCCAGGACGTGCAAACAGTATAAGTTTACGCAGACTGATGTCCCTGTCAAATACACCGAGAAACACCAGAAACACAATTATAAGGGTGAGCGATTGGTGGATAGATGTCGGGC
>Rifleman
TAGCATTTCTCAGATAAACCATGTGCAATGCACGAATGTATGGCACCTTCGATTTCGAGTCTTTCCCGCTGCTCTCTTACCAAACCTCCTAGGGCGTGTATACAGTGTAAGTTTAAGCAGACTGATGTCCCTGTCAAATACACTGAGAGACGCCAGAAAAAAGATCATGAGGGTGAGCCAATGGTCGATAGATTTCAAGC
>Chimney-swift
AAGCATCTCACAGATAAACCATGTTCACTAAGCGAATGTATCATAGCTTCGATTTCTAATTTTTTTCGCTCCTCTTTTACCGAACCTCCCAAGACGTTTAAACTGTGTAATTTAAAGCAGACCGATGTCCCTGTCAGATACACTGAGTAACACCGGAAACAAGATTATGGGGGTGATCTAATGGTAGATAGAGTAGGGGC
>Budgerigar
TAGCATTTCACAGATAAACTATGTTCAATCCGCGAATGTAGGATAGCTTCAATTTCGAGTCTTTTCCGTTCCTCTCTTACCAAAGCCCCTAGGACGTGTATACGGTCTAAGTTGAAGCAAACCGATGTCCTTGTCAAATATATTGAAAAAAACCGGAAACGAGATTACGAGGGTGATCCAATGGTAGATAGGTTTCGGGC
>Yellow-throated-sandgrouse
TAGCATCTCACAGATAAACTATGGTCATTGTGCGGATGTATGATAGGTTCGATTACGAGTCTTGTCCGCTCCGCTCTTACCAAAACTCTTAGGACGTGTAACCTGTGTAAGTTGAAGACGACTGATGTCCCTGTCAAATATACTGAGTAACACCGGAAACAAGATTATGAGGGTGATGCAATGGTAGATAGAGTACGGGC
>Bar-tailed-trogon
TAGCACCTTACAGATAAACTACGTTCATTGCGCGAATGTATATTAGCTTCGATTTCGAGTCTATTCCGCTCCTATCTTACGAAACTCCCTAGGACGTGTAAACTATGTAAGTTGAAGCAGACTGACGCCCCCGTCAAATATACTAAGTGATACCAGAAACAAGATTACGAGGATGATCCAAGCGTAGATAGATTTCGGGC
>Annas-hummingbird
TAGCATCTCGCAGATAAGCTATGTTGACTGCGCGAATGTATGATCGCTTCGATTTCGAGTCTTTTCCGCTCCCCCCTTACCAAACCTTCCAGGACGTCTAAACTGTGTAAGTTGAAGCAGACTAATGTCCCTGAAAAATCTATTGAGTTACACCGGAAACAAGATTATGGGGGTGATCCAATGGTAGATTGAATCAGTGC
>Common-cuckoo
TAGCATCTTATAGATAAACTATGTTCAGTGCGCGAATATATGATAGCTTCGATTTCGTGTCTTTTCTGCTCATCTCTTGCCAAACCTCCTAGGACGTATCAACTGGGTAAGTCGGAGTAGACTAGTGTCCATCTCAAATATACTGAGTAACATCGGAAACAAGATTATGAGGATGATCCAATGGGAGATAGATTTCAGGC
>Great-crested-grebe
TAGCATCTTACAGATAAACTATGTTCATTGCGCAAATATATGATAGCTTCGACTTCGAGTCTTTTCCGCTCCTCCCTTACCAGACCTCCTAGGACGTGTAAACTGTGTAAGTTGAAGCAGGCCGCTGTCCCTGTTAAATATACGGGGTAACACCGGAGATAAGATTATCAGGGTGATCCAATGGTAGATTTATCTCGCGC
>MacQueens-bustard
TAGTATATCACAGAGAAACTATGTTCATTGCGCGAATGTATGATAGCTTCGATTTCGAATGTTTTCCGCTCGTCTCTTACCAAACCTCCTAGGACATGTAAACTGCGTAAGTTGAAGCAGACTGATGTCCCTGTCAAATATACTGAGTAACACCCGAAACAAGATTATGAGGGTGATTCAATGCTAGATAGATTTCCGGC
>Pekin-duck
TAAAATCTCACATATATGCTGTGTCCATTGCGCGAATGTATGTAAGCTTCGTTTTCGAGTCTTTCCCTCTCCTCTCTCACCAAACCCTCTAGAACGTGTGAACTGTGTAAGTTGAAGAAGACTAATGTCCCCGTCAAATATAATGATTAACTCCGGAAGCAAGGTTATGAAGGTGATCGAAAAATAGATAGATTTTGGGC
>Barn-owl
TAGCATCTCACAGATACACTATGTTCATTGCGCGAATGTATGATAGCTTCGATTTCGAGTCTTTTCCACCCCTCTCTTACCAAACCCCCTAGGACGTGTAAACTGTGTAAGTTGAAGCAGACTGATGTCCCTGTCAAATATACTGAGTACCACCGGAAACAGGATTATGAGGGTGATCCAATGGTAGATAGATTTCGGGC
>American-flamingo
TAACATCGCACAGATAAACTATGTTCATTGCGCAAATGTATGATAGCTTCGATTTCGAGTCTTTTCCGCTCCTCTCTTACCAGACCTCCTAGGACGAGTAAACTGTGTAAGTTGAAGCAGACCGATGTCCCTGTCAAATATACTGGGTAACACCGGAAACGAGATTATAAGGGTGATCCAATGGTAGATAGATTTCGGGC
>Downy-woodpecker
TACCATCTCACAGATAAACTATGTTCATTGTACGAATGTATGATAGCTTCAATTTCGAGTCTTTTCCGCTCTTCTCTGACCAGACCCACAAGGAAATGTAAACTGTGTAAGTTTAAGCAGAATGATGTCCCTTTCAAATGTACGGAGTAACAGCAGAAATAAGGTTACGAGGGTGATCCGCCGGTAGACAGATTTTGGGC
>Chicken
GAGAGTTTCACAGGTGGACTATGTCAGGTGCGCGAATGTATAAGAGGTTCGATTTCAAGTCTTTTCCGCCCTTTTCTTACCAAACCTTATAGTACGCGTGAAATGTGTAAGTTGAGGCAGACAAATGTCCTCGTCAGATATACTGGGTAACTCTGGAAGCAAAATTATCAAGGTACTCGAAAGATAGATAGATTTTGAGC
>Hoatzin
TAGCATCTCACAGGTAAACTATGCTCATTGCGCGAATGTATGATAGCTTCTACTTCGAGTCTTTTCCGCTCCTCTCTTACCAAACCTCATAGGACGTGTAAATTATGTAAGTTGATGCAGATTGATGTCCCTGTCAAATATACTGAGTTACACCTGGGGCAACATTATAAGGGTGATTCAACGGTAGATAGATTTCGGGC
>Carmine-bee-eater
TAGCATCTCACAGATACACTATTTCCATTGCGCGAATGTATGATAGTTTCGATTTCGAGTCTTTTCCGCTCCTCTCCTAGCAGACCCCTTAGGACGTGTAAACTGTGTAAATTAAAACAGACTGGTGACCCTGTCACATATGCTGAGTAACACCGGAAACTAGATTATGGGGGTGAACCAATAGTAGGTAGATCTCGGGC
>Rhinoceros-hornbill
TAGCATCTTACAGATAAACTATATGCATTGCGCGAGTGTATGATAGCTTCAATTTCGAGTCTTTTCCGCTCCTCTCTTACCGAGCCCCCTAGAACGTGTAAGCTGTGTAAGTCGAAGCAGCCTTATGTCTCTGTCAAATATACTGAGTAACACCGGAAACAAGATCACGAGGGTGATCCAATGGTAGATAGATTTCCGGC
