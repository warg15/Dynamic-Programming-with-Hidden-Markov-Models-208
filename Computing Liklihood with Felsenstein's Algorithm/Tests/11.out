-3505.898264331048
